docs/suspensive.org/src/pages/docs/react/ErrorBoundary.en.mdx

```diff
@@ -364,6 +365,226 @@ export const ErrorAfter2s = () => {
 
 </Sandpack>
 
+### props.shouldCatch
+
+shouldCatch determines whether `<ErrorBoundary/>` should catch errors based on conditions.
+
+It accepts three criteria: Boolean, Constructor, and Callback, and defaults to true.
+
+<Tabs items={['Boolean', 'Constructor', 'Callback']}>
+  <Tabs.Tab>
+  
+```tsx {10}
```

Comment:
I think {10} is good, but /shouldCatch/ also could be same benefit and good to maintain 

```suggestion
```tsx /shouldCatch/
```

Reply:
oh, I think it's definitely better. 👍